tests/manual/ipc/TestIPC.h

```diff
@@ -27,5 +27,17 @@
 ** SPDX-License-Identifier: MIT
 **
 **********************************************************************/
+//! [indoc]
```

Comment:
Could you move that down, after "#pragma once" (which is not needed in the snippet)?